transformer_engine/pytorch/te_onnx_extensions.py

```diff
@@ -157,6 +166,79 @@ def onnx_fp8_relu(g, inputs, scale, amax, scale_inv, fp8_tensor, otype):
     return relu
 
 
+@_onnx_symbolic("aten::glu")
```

Comment:
Here and in the other sym funcs below you are registering all the functions using the same name: "aten::glu". This is not required and will mask the correct sym func for `aten::glu`.

The code at the end of the file registers these sym functions correctly using `register_custom_op_symbolic`